### DISCUSSION TITLE
Storage configuration failed  Errors in installing the system via ISO

### DISCUSSION THREAD

**Discussion by ldceven** [Security-Onion-Solutions/securityonion#1879] [Unsupported Versions] [answered]
Hi!
My server test environment:
Installed Security Onion Version 16.04.7.1  before, now install Security Onion Version 2.3.2
Source: ISO
/dev/sda 300G: install os
/dev/sdb 1.7T: nsm storage
Installing Security Onion Version 2.3.2 encounters installation error problems, the error message is as follows:
"Storage configuration failed: Failed to find a suitable stage1 device: EFI System Partition cannot ..."

Have you ever encountered the same problem? How to deal with it? Thank you for your help.
![2481605091237_ pic](https://user-images.githubusercontent.com/67267374/98802144-cb7d4b80-244d-11eb-8ff6-78dcba32fa59.jpg)

**Comment by TOoSmOotH** [comment]
Try legacy boot mode. What type of storage are you using?

**Reply by ldceven** [reply]
Okay, I will try to use other guidance methods. Used Dell R730xd server, storage type is HDD

**Reply by ldceven** [reply]
Hello, after I changed the boot mode to bios mode, there was no storage error prompt during the installation process, but it was stuck on this interface for more than 1 hour. Is it normal?
![WechatIMG251](https://user-images.githubusercontent.com/67267374/99041084-3ac97b80-25c5-11eb-9de2-45c5c7878ffe.jpeg)

**Comment by TOoSmOotH** [comment, marked as answer]
EFI mode is now fixed in 2.3.10
